Dockerfile

```diff
@@ -194,6 +196,7 @@ ENV ADMINPASS=
 ENV ADMINMAIL=
 
 VOLUME [ "/opt/kimai/var" ]
+WORKDIR /opt/kimai
```

Comment:
why this?  i thought we have written each command with absolut paths?

Reply:
Doesn't do any harm does it?  Plus now commands like 

    docker exec -ti kimai tail -f var/log/dec.log

Does not need to be pathed

Reply:
Every single line in the dockerfile is its own layer - this will increase the image size and is not necessary at this point - imho

Reply:
ok, change made